### PR TITLE
Automatic mock files generator

### PR DESCRIPTION
**Pull request by CSenshi**
For Issue #28:

Following PR adds ability to create mock files with bash script (mocker.sh). Anyone willing to add new destination and interface for mocker to generate can got to script and add them into the given array.
Also added mocker.sh to makefile

### CODE REVIEW

File: mocker.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Github Repository Link
+GH_REPO=github.com/bakurits/mattermost-plugin-anonymous
+
+# Array of (destination,interface) tuples
+# If new element needs to be added, please delimit destination and interface by comma like other elements
+array=(
+    anonymous,Anonymous
```
Comment (bakurits) [inline mocker.sh:9]: I think extra space between will make that more readable

File: mocker.sh [hunk 2]
```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Github Repository Link
+GH_REPO=github.com/bakurits/mattermost-plugin-anonymous
+
+# Array of (destination,interface) tuples
```
Comment (bakurits) [inline mocker.sh:6]: `package` will be more suitable than `destination`

File: mocker.sh (hunk 1 above)
Comment (CSenshi) [reply mocker.sh:9]: Space is invalid solution as it parses using space. Any delimiter will do but not space,enter or tab. We can choose delimiter which won't occur in file names or interfaces

File: mocker.sh (hunk 2 above)
Comment (CSenshi) [reply mocker.sh:6]: Right, package is better name

File: mocker.sh (hunk 1 above)
Comment (dbejanishvili) [reply mocker.sh:9]: Then a colon would be better

Comment (CSenshi) [reply mocker.sh]: done

**Pull request merged**